tests/test_simple_sample_fps.py

```diff
@@ -0,0 +1,60 @@
+import unittest
+
+from sklearn.datasets import load_boston
+from sklearn.utils.validation import NotFittedError
+
+from skcosmo.sample_selection import SimpleFPS
+
+
+class TestSimpleFPS(unittest.TestCase):
+    def setUp(self):
+        self.X, _ = load_boston(return_X_y=True)
+        self.idx = [0, 410, 492, 102, 134, 413, 54, 32, 353, 126]
```

Comment:
Same comment as in the simple CUR test regarding the indices

Reply:
Same with this! Without an initial point selected, it'll choose the 0th index first, then a deterministic set afterwards.